wrappers/wrapper.rb

```diff
@@ -1497,8 +1503,8 @@ def simplify_service_setup_duration_and_vehicle_setup_modifiers(vrp, result = ni
             # then the following logic needs to be updated. Basically we need to do each_with_index
             # and apply the setup duration increment to every pair except index == point.matrix_index
             # even if they were 0 in the first place.
-            matrix.time.each{ |row|
-              row[point.matrix_index] += (coef_setup * setup_duration + additional_setup).to_i if row[point.matrix_index] > 0
+            matrix.time.each.with_index{ |row, row_index|
```

Comment:
Now that it became true, shall we delete the warning? 
```suggestion
            matrix.time.each_with_index{ |row, row_index|
```